includes/class-wc-order.php

```diff
@@ -766,6 +766,11 @@ public function get_address( $type = 'billing' ) {
 	 */
 	public function get_shipping_address_map_url() {
 		$address = apply_filters( 'woocommerce_shipping_address_map_url_parts', $this->get_address( 'shipping' ), $this );
+		foreach ( array( 'first_name', 'last_name', 'company' ) as $key ) {
```

Comment:
It would be better to get the address, remove the elements, and THEN filter, so user has full control over final array of data.